pyaedt/edb_core/edb_data/simulation_setup.py

```diff
@@ -0,0 +1,701 @@
+        self._edb_sweep_data.UseQ3DForDC = value
+        self._update_sweep()
+
+    @pyaedt_function_handler()
+    def _set_frequencies(self, freq_sweep_string="Linear Step: 0GHz to 20GHz, step=0.05GHz"):
+        self._edb_sweep_data.SetFrequencies(freq_sweep_string)
+        self._update_sweep()
+
+    @pyaedt_function_handler()
+    def set_frequencies_linear_scale(self, start="0.1GHz", stop="20GHz", step="50MHz"):
+        """Set a linear scale frequency sweep.
+
+        Parameters
+        ----------
+        start : str, float
+            Start frequency.
+        stop : str, float
+            Stop frequency.
+        step : str, float
+            Step frequency.
+
+        Returns
+        -------
+        bool
+            ``True`` if correctly executed, ``False`` otherwise.
+
+        """
+        self._edb_sweep_data.Frequencies = self._edb_sweep_data.SetFrequencies(start, stop, step)
+        return self._update_sweep()
+
+    @pyaedt_function_handler()
+    def set_frequencies_linear_count(self, start="1kHz", stop="0.1GHz", count=10):
+        """Set a linear count frequency sweep.
+
+        Parameters
+        ----------
+        start : str, float
+            Start frequency.
+        stop : str, float
+            Stop frequency.
+        count : int
+            Step frequency.
+
+        Returns
+        -------
+        bool
+            ``True`` if correctly executed, ``False`` otherwise.
+
+        """
+        start = self._sim_setup._pedb.arg_to_dim(start, "Hz")
+        stop = self._sim_setup._pedb.arg_to_dim(stop, "Hz")
+        self._edb_sweep_data.Frequencies = self._edb_sweep_data.SetFrequencies(start, stop, count)
+        return self._update_sweep()
+
+    @pyaedt_function_handler()
+    def set_frequencies_log_scale(self, start="1kHz", stop="0.1GHz", samples=10):
+        """Set a log count frequency sweep.
+
+        Parameters
+        ----------
+        start : str, float
+            Start frequency.
+        stop : str, float
+            Stop frequency.
+        samples : int
+            Step frequency.
+
+        Returns
+        -------
+        bool
+            ``True`` if correctly executed, ``False`` otherwise.
+        """
+        start = self._sim_setup._pedb.arg_to_dim(start, "Hz")
+        stop = self._sim_setup._pedb.arg_to_dim(stop, "Hz")
+        self._edb_sweep_data.Frequencies = self._edb_sweep_data.SetLogFrequencies(start, stop, samples)
+        return self._update_sweep()
+
+    @pyaedt_function_handler()
+    def set_frequencies(self, frequency_list=None):
+        """Set frequency list to the sweep frequencies.
+
+        Parameters
+        ----------
+        frequency_list : list, optional
+            List of lists with four elements. Each list must contain:
+
+              1- frequency type (``"linear count"``, ``"log scale"`` or ``"linear scale"``)
+              2- start frequency
+              3- stop frequency
+              4- step frequency or count
```

Comment:
```suggestion
            List of lists with four elements. The default is ``None``. If provided, each list must contain:

              1 - frequency type (``"linear count"``, ``"log scale"``, or ``"linear scale"``)
              2 - start frequency
              3 - stop frequency
              4 - step frequency or count
```
What is the behavior that occurs when ``None`` is used. Describe in the second sentence like this: The default is ``None``, in which case ....."